arbitrum-docs/for-devs/third-party-docs/MetaMask/metamask.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'MetaMask Smart Accounts'
+description: 'Learn about MetaMask Smart Accounts and how to use them with Arbitrum'
+sidebar_label: 'MetaMask'
+third_party_content_owner: 'alexandratran'
+---
+
+The [MetaMask Delegation Toolkit](https://docs.metamask.io/delegation-toolkit/) is a collection of tools for embedding
+MetaMask Smart Accounts into dapps.
+[Smart accounts](https://docs.metamask.io/delegation-toolkit/concepts/smart-accounts/) support programmable account
+behavior and advanced features like delegated permissions, multi-signature approvals, and gas abstraction.
+Unlike traditional wallets, which rely on private keys for every transaction, MetaMask Smart Accounts use smart contracts to
```

Comment:
Maybe we can remove this sentence as this explains what are smart accounts, or maybe just change the order?